src/draco/io/stl_decoder_test.cc

```diff
@@ -72,12 +72,15 @@ class StlDecoderTest : public ::testing::Test {
 TEST_F(StlDecoderTest, ValidBinarySTL) {
   test_decoding("teapot.stl", 946);
   test_decoding("40mmcube.stl", 12);
+  test_decoding("square_15_15_250.STL", 12);
```

Comment:
Can you add a test with binary stl file that starts with the 80 bytes:

"solid q\nfacet normal 0 0 0\nouter loop\nvertex 0 0 0\nvertex 1 1 1\nvertex 1 1 0\n\n\n\n"

? This seems a bit silly, but so does a binary stl file that starts with "solid square_15_15_250                                                          ".

Reply:
Done.  Fair warning, I'm kind of sick.  While attempting to spell "through" I wrote "throw" so give it a bit of a close read.